spring-cloud-gcp-docs/src/main/asciidoc/datastore.adoc

```diff
@@ -380,6 +404,31 @@ public class MyApplication {
 
 ----
 
+==== Transactions
```

Comment:
I think it would make sense to consolidate all transaction-related documentation into one section instead of having two.

Reply:
There was a concerted effort to lay out the structure of ref doc for our Spring Data modules back when we did Spanner, and we have "===" level sections Repository and Template since they are rather large concepts and then "====" sections for features under them (Transactions being such a section). 

If we were to consolidate move the Transactions section out of the Repository and Template sections, it would have to be a "===" section, and I don't think it's a concept that stands at the same level as those two. Furthermore, the transactions methods described specifically allow usage of Repository and Template objects within the anonymous function, and I think moving them away from those parent sections would be kind of orphaning those mentions within the Transactions section.